sql/src/test/java/org/apache/druid/sql/calcite/CalciteInsertDmlTest.java

```diff
@@ -1765,7 +1762,6 @@ public void testErrorWhenInputSourceInvalid()
                              + "partitioned by DAY\n"
                              + "clustered by channel";
     HashMap<String, Object> context = new HashMap<>(DEFAULT_CONTEXT);
```

Comment:
I can't comment on the `testInsertWithInvalidColumnNameInIngest` testcase; but the intention of the check is to ensure that something like:
```
INSERT INTO t SELECT __time, 1+1 FROM foo PARTITIONED BY ALL
```
is catched ; could you change or add something like this as a testcase?